ui/src/App.scss

```diff
@@ -98,6 +98,11 @@
   }
 }
 
+.ant-tooltip a {
+  color: white;
```

Comment:
Could you put a screenshot for @StellaCh so that we're sure that's the color we want?